red/MyDate/lib/MyDate.pm

```diff
@@ -0,0 +1,169 @@
+package MyDate;
+
+use v5.16;
+use strict;
+use warnings;
+use Carp;
+our $AUTOLOAD;
+
+=head1 NAME
+
+MyDate - Yet another simple utility for dates
+
+=head1 VERSION
+
+Version 0.01
+
+=cut
+
+our $VERSION = '0.01';
+
+
+=head1 SYNOPSIS
+
+Simple OO interface onto localtime.
+
+    use MyDate;
+
+    my $date = MyDate->new();
+    my ( $day, $month, $year ) =
+      $date->date, $date->month, $date->year; # e.g. 26 4 2015
+    ...
+
+=head1 SUBROUTINES/METHODS
+
+=head2 new
+
+Get an instance of MyDate. Not a whole lot to configure here.
+
+=cut
+
+sub new {
+  bless \my $self, $_[0]
```

Comment:
I disagree, largely because the convention for other ref types is just the same (`my $h = \%h`, `my $cb = \&do_foo`). Also, the places I've seen this idiom used before specify it this way (e.g. [List::Util::reduce](https://chromium.googlesource.com/chromium/deps/perl/+/2693003f205ef0f4429a426cc8f6d19a9596e67d/perl/lib/List/Util/PP.pm#34)), and also, this is a pretty simple statement, so I don't agree that it needs any kind of operator disambiguation.